src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -1,87 +1,24 @@
-import React, { useState, useMemo, useRef, useCallback } from "react";
-import { useHistory } from "react-router-dom";
-import { OverlayTrigger, Popover } from "react-bootstrap";
+import React, { ReactNode } from "react";
 
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faTicketAlt } from "@fortawesome/free-solid-svg-icons";
+import { currentVenueSelectorData, parentVenueSelector } from "utils/selectors";
 
-import firebase from "firebase/app";
-
-import { DEFAULT_PROFILE_IMAGE, PLAYA_VENUE_ID } from "settings";
-import { IS_BURN } from "secrets";
-
-import { UpcomingEvent } from "types/UpcomingEvent";
-
-import {
-  currentVenueSelectorData,
-  parentVenueSelector,
-  radioStationsSelector,
-} from "utils/selectors";
-import { hasElements } from "utils/types";
-import { venueInsideUrl } from "utils/url";
-
-import { useRadio } from "hooks/useRadio";
 import { useSelector } from "hooks/useSelector";
-import { useUser } from "hooks/useUser";
-import { useVenueId } from "hooks/useVenueId";
 import { useFirestoreConnect } from "hooks/useFirestoreConnect";
 
-import { GiftTicketModal } from "components/organisms/GiftTicketModal/GiftTicketModal";
-import { ProfilePopoverContent } from "components/organisms/ProfileModal";
-import { RadioModal } from "components/organisms/RadioModal/RadioModal";
-import { SchedulePageModal } from "components/organisms/SchedulePageModal/SchedulePageModal";
-
-import NavSearchBar from "components/molecules/NavSearchBar";
-import UpcomingTickets from "components/molecules/UpcomingTickets";
-import { VenuePartygoers } from "components/molecules/VenuePartygoers";
-
-import { NavBarLogin } from "./NavBarLogin";
-
 import "./NavBar.scss";
-import * as S from "./Navbar.styles";
 import "./playa.scss";
```

Comment:
Is this `playa.scss` imported from any other files? If not, can we maybe combine it with `NavBar.scss`?

Reply:
It is being used in one more file but I think this can be removed in the future refactoring PR.

Reply:
Oh? Which file out of curiosity? (when I looked at it kat I thought it was only this one, but there are 2 separate playa.scss files in the codebase, so you’d need to check where they are being imported from; if my memory is correct)

Reply:
Actually it's Playa.tsx which is not used anywhere.

Reply:
That's what I was referring to above, they are different files, just named the same.

`Playa` imports `src/components/templates/Playa/Playa.scss`

`NavBar` imports `src/components/molecules/NavBar/playa.scss`

The `NavBar` one is only used by the `NavBar`, and should be renamed `NavBar.scss` or similar.

Reply:
@denisdimitrov This still seems like it needs to be done?